.github/workflows/boot-kotlin.yml

```
@@ -17,23 +17,19 @@ jobs:
   build:
 
     runs-on: ubuntu-latest
+    default:
```

actionlint (1.7.12)

[error] unexpected key "default" for "job" section. expected one of "concurrency", "container", "continue-on-error", "defaults", "env", "environment", "if", "name", "needs", "outputs", "permissions", "runs-on", "secrets", "services", "snapshot", "steps", "strategy", "timeout-minutes", "uses", "with"

(syntax-check)